client/app/assets/less/redash/query.less

```diff
@@ -245,11 +245,6 @@ a.label-tag {
     align-content: space-around;
     padding: 0;
     overflow-x: hidden;
-
-    .pivot-table-visualization-container > table,
-    .visualization-renderer > .visualization-renderer-wrapper {
-      overflow: visible;
```

Comment:
How about rule for `.pivot-table-visualization-container`? Was there any issue as well, or this rule no longer needed?

Reply:
For pivot I think this rule was outdated since its migration to React. I quickly tested it and didn't see any issues with it.